fastlane/spec/plugins_specs/plugin_info_collector_spec.rb

```diff
@@ -216,6 +250,8 @@
 
       info = Fastlane::PluginInfo.new('test_name', 'Fabricio Devtoolio', 'fabric.devtools@gmail.com', 'summary')
 
+      expect(FastlaneCore::Helper).to receive(:backticks).with('git config --get user.email').and_return('')
+      expect(FastlaneCore::Helper).to receive(:backticks).with('git config --get user.name').and_return('')
```

Comment:
nitpick, but I'd group these up with the other set of `expect`s that are setting up mocks above